application/common/config/main.php

```diff
@@ -225,5 +221,10 @@
         'mfaManagerHelpBcc'             => Env::get('MFA_MANAGER_HELP_BCC', ''),
         'contingentUserDuration'        => Env::get('CONTINGENT_USER_DURATION', '+4 weeks'),
         'inviteEmailDelaySeconds'       => Env::get('INVITE_EMAIL_DELAY_SECONDS', 0),
+        'hibpCheckOnLogin'              => Env::get('HIBP_CHECK_ON_LOGIN', true),
+        'hibpCheckInterval'             => Env::get('HIBP_CHECK_INTERVAL', '+1 week'),
+        'hibpGracePeriod'               => Env::get('HIBP_GRACE_PERIOD', '+1 week'),
+        'hibpTrackingOnly'              => Env::get('HIBP_TRACKING_ONLY', false),
```

Comment:
Do you have a process in mind to ensure that we set this to `true` when it goes live? (That's what you said Sheryl wants, right?)

Reply:
Yeah, we'll have to run through TF updates first and add the variable. Since it is a temporary thing that we may not use for all IdPs I defaulted to `false` the way we want it to really work.